pkg/config/load.go

```diff
@@ -39,6 +39,8 @@ type Prowgen struct {
 	Rehearsals Rehearsals `json:"rehearsals,omitempty"`
 	// SlackReporterConfigs defines all desired slack reporter info for included jobs
 	SlackReporterConfigs []SlackReporterConfig `json:"slack_reporter,omitempty"`
+	// SkipPresubmitsGeneration allow users to skip the presubmit generation for that specific variant
+	SkipPresubmitsGeneration SkipPresubmitsGeneration `json:"skip_presubmits_generation,omitempty"`
```

Comment:
This is more general now, since you allow the user to define the branch and the variant. The whole generation should respect this, then, not only when the operator stanza is found.